avocado/core/remote/runner.py

```diff
@@ -325,8 +329,8 @@ class VMTestRunner(RemoteTestRunner):
     Test runner to run tests using libvirt domain
     """
 
-    def __init__(self, job, result_proxy):
-        super(VMTestRunner, self).__init__(job, result_proxy)
+    def __init__(self, job, result_proxy, result):
```

Comment:
^^ In my opinion a patch should be standalone change. There could be intermittent patches to improve readability of the change, but then they should be part of one patch set.